modules/proxyconnection/CustomSignalingLayer.js

```diff
@@ -58,6 +48,34 @@ export default class CustomSignalingLayer extends SignalingLayer {
         return this.ssrcOwners.get(ssrc);
     }
 
+    /**
+     * @inheritDoc
+     */
+    getTrackSourceName(ssrc) { // eslint-disable-line no-unused-vars
+        return undefined;
```

Comment:
Since `setTrackSourceName` is an empty function maybe we should do this an empty function as well.

I don't have any preferences whether it will be empty or `return undefined`. Lets be consistent.

Reply:
Actually, it is consistent with the rest of the class. All the other methods return `undefined` for getters and since `setTrackSourceName` is a setter, it does nothing.

Reply:
BTW, I just did an alpha sort of the methods, this is how it is was before :)